frontend/index_es.html

```diff
@@ -0,0 +1,551 @@
+                id="ThunderstoneSearchForm">
+
+              <input type="hidden" name="pr" value="metanrel" />
+              <div class="form-group">
+                  <label for="query" class="sr-only">Buscar NREL.gov</label>
+                  <input
+                      type="search"
+                      id="query" name="query"
+                      maxlength="2047"
+                      placeholder="Buscar NREL.gov"
+                      class="form-control"
+                      aria-label="Buscar NREL.gov">
+                  <button type="submit" class="btn btn-default">Buscar</button>
+              </div>
+            </form>
+          </div>
+        </nav>
+      </div>
+    </div>
+    </div>
+    </div>    
+    <!-- End collapsible site search bar -->
+    <!--startindex-->
+
+		<div class="container" id="content" role="main">
+         <div class="row">
+            <div class="col-lg-9">
+               <h1>Universidad de Puerto Rico Recinto de Mayagüez Proyecto NSF CIVIC</h1>
+               <!--<h1 id="studyTitleOne"></h1>--> <!--This was removed do to lack of translated token at .json file. This is something that we will have to discuss with Shankari.-->
+               <p class="lead">Gracias por participar en la Universidad de Puerto Rico Recinto de Mayagüez Proyecto NSF CIVIC</p>
+               <!--<p class="lead">Gracias por participar en el <a id="studyTitleTwo"></a>.</p>--> <!--This was removed do to lack of translated token at .json file. This is something that we will have to discuss with Shankari.-->
+               
+               <h2>Propósito del estudio</h2>
+
+               <p id="shortTextualDescription"></p>
+               
+               <h2>Recolección de datos</h2>
+               
+               <p>Utilizaremos la plataforma de <a href="https://www.nrel.gov/transportation/openpath.html">NREL OpenPATH</a>
+                para la recolección de datos. Esta plataforma de código abierto recoge información completa 
+                de sus viajes y la utiliza para estimar su huella de carbono en el transporte 
+                y compararla con los objetivos de EE.UU. para 2030 y 2050. También calcula y 
+                publica métricas agregadas sobre la moda y la distancia recorrida.
+
+               </p>
+
+               <div class="row justify-content-center">
+                     
+                 <div class="col-md-6 mb-4"><img class="img-responsive img-fluid" src="client/img/OpenPATH-dashboard-preview.png" alt="OpenPATH dashboard preview"></div>
+                 <div class="col-md-6 mb-4"><img class="img-responsive img-fluid" src="client/img/public-dashboard-preview.png" alt="Public dashboard preview"></div>
+
+               </div>
+               
+               <p id="whyWeCollect"></p> <!--This was removed do to lack of translated token at .json file. This is something that we will have to discuss with Shankari.-->
```

Comment:
This is not actually removed 😄  And it is also filled in with the javascript.
https://github.com/e-mission/nrel-openpath-join-page/pull/2/files#diff-f9270d52d0bfede95867823f07561184cca02839279f355dea172edfbd29a6e8R522

```
            const WHY_WE_COLLECT_TEXT = `${data.intro.translated_text.es.why_we_collect_es}`;
            $('#whyWeCollect').text(WHY_WE_COLLECT_TEXT);
```

Reply:
Yes, we accidentally left that comment in the code 😆. We will remove this for our next PR, in which hopefully it will be sent by tomorrow (12/25/22) early in the morning.